.circleci/config.yml

```diff
@@ -93,6 +117,12 @@ workflows:
       - test-bs:
           requires:
             - build
+      - generate-repo:
+          requires:
+            - build
+          filters:
+            branches:
+              only: westbrook/generate-repo
```

Comment:
We'd want this just on `master`, of course, but is there any other gating we'd want to use to reduce incidences of this?

Reply:
Is it possible to do it only when `create` has a version bump?

Reply:
This is a good question that I'm not 100% sure about. It might be possible to do this with Lerna/Convential Commits... @daKmoR any experience with this? It might be possible to leverage `postpublish`?

Reply:
Also, if the output is the same, it won't be able to commit anything, so nothing gets pushed: https://app.circleci.com/pipelines/github/open-wc/open-wc/6152/workflows/b7f8bba8-86ec-47d8-bfa7-60cbb469835f/jobs/9037/parallel-runs/0/steps/0-107